regolith/schemas.py

```diff
@@ -2291,7 +2291,7 @@
         },
         "day": {
             "description": "day that the proposal was submitted",
-            "required": True,
+            "required": False,
```

Comment:
I had to get rid of the requirement for day because it would mess up the grant's dates. The grant or linking proposal can have either a begin_day or day but if there is both, the begin_day (and the begin_month and begin_year) similarly get overwritten.  I made the field not required and removed it because if the proposal has both it gives the wrong date for the grant.

Reply:
Good catch, thanks!